### PR TITLE
Storage Blobs Readme [wip]

### PR DESCRIPTION
**Pull request by kristapratico**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/storage/azure-storage-blob/README.md [hunk 1]
```diff
@@ -1,15 +1,217 @@
...
+### Authenticate the client
+
+Interaction with Storage Blobs starts with an instance of the BlobServiceClient class. You need an existing storage account, its URL, and a credential to instantiate the client object.
+
+#### Get credentials
+
+To authenticate the client you have a few options:
+1. Use a SAS token string 
+2. Use an account shared access key
+3. Use a token credential from azure.identity
```
Comment (annatisch) [inline sdk/storage/azure-storage-blob/README.md]: We should make a note to upadte "azure.identity" to a link to that package once it's published on PyPI (hopefully monday)

File: sdk/storage/azure-storage-blob/README.md [hunk 2]
```diff
@@ -1,15 +1,217 @@
...
+### Uploading a blob
+Upload a blob to your container.
+
+```python
+from azure.storage.blob import BlobServiceClient
+
+service = BlobServiceClient.from_connection_string(conn_str="my_connection_string")
+container = service.get_container_client(container="mycontainer")
+
+blob = container.get_blob_client("my_blob")
```
Comment (annatisch) [inline sdk/storage/azure-storage-blob/README.md]: We can simplify the sample by instantiating the blob client directly:
```
blob = BlobClient.from_connection_string("my_connection_string", container="mycontainer", blob="my_blob")
```

File: sdk/storage/azure-storage-blob/README.md [hunk 3]
```diff
@@ -1,15 +1,217 @@
...
+### Downloading a blob
+Download a blob from your container.
+
+```python
+from azure.storage.blob import BlobServiceClient
+
+service = BlobServiceClient.from_connection_string(conn_str="my_connection_string")
+container = service.get_container_client(container="mycontainer")
+
+blob = container.get_blob_client("my_blob")
```
Comment (annatisch) [inline sdk/storage/azure-storage-blob/README.md]: We can simplify the sample by instantiating the blob client directly:
```
blob = BlobClient.from_connection_string("my_connection_string", container="mycontainer", blob="my_blob")
```

**Pull request merged**